client.go

```diff
@@ -0,0 +1,22 @@
+package bacnet
```

Comment:
Do not put the implementation in the root. Rather follow [standard Go project layout](https://github.com/golang-standards/project-layout)